docs/03/a.md

```diff
@@ -12,21 +12,22 @@ formatted in upper camel case with:
 
 The formatting options are:
 
-    upper    | uppercase       : all uppercase letters
-    lower    | lowercase       : all lowercase letters
-    cap      | capitalize      : uppercase first letter
-    decap    | decapitalize    : lowercase first letter
-    start    | start-case      : uppercase the first letter of each word
-    word     | word-only       : remove all non-word letters (only a-zA-Z0-9_)
-    space    | word-space      : replace all non-word letters (only a-zA-Z0-9) with a whitespace
-    Camel    | upper-camel     : upper camel case (start-case, word-only)
-    camel    | lower-camel     : lower camel case (start-case, word-only, decapitalize)
-    hyphen   | hyphenate       : replace spaces with hyphens
-    norm     | normalize       : all lowercase with hyphens (lowercase, hyphenate)
-    snake    | snake-case      : replace spaces and dots with underscores
-    package  | package-naming  : replace spaces with dots
-    packaged | package-dir     : replace dots with slashes (net.databinder -> net/databinder)
-    random   | generate-random : appends random characters to the given string
+    upper      | uppercase            : all uppercase letters
+    lower      | lowercase            : all lowercase letters
+    cap        | capitalize           : uppercase first letter
+    decap      | decapitalize         : lowercase first letter
+    start      | start-case           : uppercase the first letter of each word
+    word       | word-only            : remove all non-word letters (only a-zA-Z0-9_)
+    space      | word-space           : replace all non-word letters (only a-zA-Z0-9) with a whitespace
+    Camel      | upper-camel          : upper camel case (start-case, word-only)
+    camel      | lower-camel          : lower camel case (start-case, word-only, decapitalize)
+    hyphen     | hyphenate            : replace spaces with hyphens
+    norm       | normalize            : all lowercase with hyphens (lowercase, hyphenate)
+    snake      | snake-case           : replace spaces and dots with underscores
+    reverseOrg | reverse-organization : tokenizes by dot and reverses the tokens (scala-lang.org -> org.scala-lang)
```

Comment:
Since there are `package` and `packaged` already, `reversePackage` maybe?

Reply:
I was thinking about the fact that it's probably enough to call it `reverse` since it doesn't replace spaces with dots like `package` does, but `reverse` can be misleading since in this case `$x;format="reverse"$` would be equal to `$x$` if `x` doesn't contain dots. Probably `dotReverse` is a better fit?

Another possible problem is verbosity:
```
$organizationName$ -> "blog.foobar.dev"
$organizationName;format="reversePackage,packaged"$ -> "dev/foobar/blog"
```

Last but not least, I can't see in the documentation any mention to the possibility to use more than one formatter for directory/file names like `$organizationName__reversePackage_packaged$`. Is it possible to use 2 or more in combination?

That said, `reversePackage` is still hugely better than `reverseOrg`, so if all the previously expressed doubts are not "blocking" I can proceed with renaming

Reply:
I just spotted an integration test with this filename into it `$test__lower,word$`. My syntax was wrong, so in fact it works. I'll update the documentation in another PR.